src/app/(sidebar)/transactions-explorer/tx/[tx]/components/TransactionDetails.tsx

```diff
@@ -0,0 +1,199 @@
+      },
+      delay: 100,
+    });
+  };
+
+  const goToTransaction: MouseEventHandler = (event) => {
+    event.preventDefault();
+    event.stopPropagation();
+
+    delayedAction({
+      action: () => {
+        endpoints.updateParams({ transaction: tx.txHash });
+        router.push(Routes.ENDPOINTS_TRANSACTIONS_SINGLE);
+      },
+      delay: 100,
+    });
+  };
+
+  useEffect(() => {
+    const parse = async () => {
+      await StellarXdr.initialize();
+
+      try {
+        const envelopeXdr = tx.envelopeXdr.toXDR().toString("base64");
+        const guesses = StellarXdr.guess(envelopeXdr);
+        const json = JsonParse(
+          StellarXdr.decode(guesses[0], envelopeXdr),
+          null,
+          parseNumberAndBigInt,
+        );
+        setXdrJson(json as Record<string, unknown>);
+        setXdr(envelopeXdr);
+
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+      } catch (e) {
+        // do nothing
+      }
+    };
+
+    parse();
+  }, [tx]);
+
+  return (
+    <Box gap="md" data-testid="explorer">
+      <PageCard heading="Transaction Envelope">
+        <Box gap="xs" align="start">
+          <InfoField
+            label="Status"
+            value={
+              <Box
+                direction="row"
+                align="center"
+                gap="xs"
+                addlClassName={tx.status.toLowerCase()}
+              >
+                <Text size="xs" as="span">
+                  {tx.status === StellarRpc.Api.GetTransactionStatus.SUCCESS
+                    ? "Successful"
+                    : "Failed"}
+                </Text>
+                <Icon.CheckCircle size="xs" color="currentColor" />
+              </Box>
+            }
+          />
+          <InfoField
+            label="Transaction"
+            value={
+              <SdsLink
+                onClick={goToTransaction}
+                href={buildEndpointHref(Routes.ENDPOINTS_TRANSACTIONS_SINGLE, {
+                  transaction: tx.txHash,
+                })}
+              >
+                <Box gap="xs" direction="row" align="center">
+                  {tx.txHash}
+                </Box>
+              </SdsLink>
+            }
+          />
+          <InfoField
+            label="Source account"
+            value={
+              <Box direction="row" gap="xs" align="center">
+                <SdsLink
+                  onClick={goToAccount}
+                  href={buildEndpointHref(Routes.ENDPOINTS_ACCOUNTS_SINGLE, {
+                    account_id: sourceAccount,
+                  })}
+                >
+                  <Asset
+                    size="sm"
+                    sourceOne={{
+                      altText: "",
+                      image: `https://id.lobstr.co/${sourceAccount}.png`,
+                    }}
+                    variant="single"
+                  />
+                  {sourceAccount}
+                </SdsLink>
```

Comment:
We can use [Profile component](https://design-system.stellar.org/profile) from SDS here.

Reply:
That's better. The one thing I noticed is that the link is blank, rather than navy blue (I copied the design from Charles' figma somewhere that Jane sent me). Not a big deal, I think.